routes/xref/update.js

```diff
@@ -16,9 +16,9 @@ module.exports.route = function route(req, res) {
     return res.send(msg);
   }
 
-  if (req.body.refs !== "refs/heads/master") {
+  if (req.body.ref !== "refs/heads/master") {
     res.status(400); // Bad request
-    const msg = "Payload was not for master, ignored it.";
+    const msg = `Payload was for ${req.body.ref}, ignored it.`;
```

Comment:
```suggestion
    const msg = `Caniuse payload was for ${req.body.ref}, ignored it.`;
```

Reply:
Alternatively, we should use a real error for console.error. Otherwise we don’t know which one failed.